### PR TITLE
improve contribute.md grammar

### PR DESCRIPTION
**Pull request by MarcoWorms**
todo: fix broken links

### CODE REVIEW

File: docs/contributing/contribute.md [hunk 1]
```diff
@@ -36,31 +36,31 @@ Use the [Feedback](https://gov.yearn.finance/c/feedback/2) section of the govern
...
+If you're a developer, we need your help! Before you begin: Familiarize yourself with Yearn, its products, and its infrastructure by reading our [Documentation](https://docs.yearn.finance/).
 
-You can see our progress on our [Github Project Board](https://github.com/orgs/yearn/projects/1) and you can immediately begin contributing where you see fit. We recommend checking out issues tagged as `help wanted` and recommend sticking to the respective repository's coding guidelines (linting, formatting, PRs, etc). It's that simple 🙂. You can also join us on the #dev channel in [Discord](https://discord.gg/w89vVxeV9h).
+You can see our progress on our [Github Project Board](https://github.com/orgs/yearn/projects/1), and you can immediately begin contributing where you see fit. We recommend checking out issues tagged as `help wanted` and recommend sticking to the respective repository's coding guidelines (linting, formatting, PRs, etc). It's that simple 🙂. You can also join us on the #dev channel in [Discord](https://discord.gg/w89vVxeV9h).
```
Comment (DarkGhost7) [inline docs/contributing/contribute.md:55]: This board is dead for a while now, maybe switch the link to here https://contribute.yearn.farm/good-first-issue.html

File: docs/contributing/contribute.md [hunk 2]
```diff
@@ -36,31 +36,31 @@ Use the [Feedback](https://gov.yearn.finance/c/feedback/2) section of the govern
...
 
 If you are interested in working on a specific project, check out the list of Yearn products in the [General Resources](#general-resources) section below or check out our list of repos [here](https://github.com/orgs/yearn/repositories).
 
-If you want an overview on how our services interact with one another in Vaults, please see the schema and accompanying descriptions in the [Vaults Overview](https://docs.yearn.finance/getting-started/products/yvaults/overview).
+If you want an overview of how our services interact with one another in Vaults, please see the schema and accompanying descriptions in the [Vaults Overview](https://docs.yearn.finance/getting-started/products/yvaults/overview).
```
Comment (DarkGhost7) [inline docs/contributing/contribute.md:59]: For an overview of how our vaults work, please see the schema ...
What `services` are we talking about here?

File: docs/contributing/contribute.md [hunk 3]
```diff
@@ -36,31 +36,31 @@ Use the [Feedback](https://gov.yearn.finance/c/feedback/2) section of the govern
...
 
-If you're wondering how **strategies** work, please check out [Yearn Vaults v2 intro](https://www.youtube.com/watch?v=C0fsYiCI54g) — hosted by our Yearn Devs. To start writing a Strategy of your own, please visit the [Yearn Brownie Mix](https://github.com/yearn/brownie-strategy-mix).
+If you're wondering how **strategies** work, please check out [Yearn Vaults v2 intro](https://www.youtube.com/watch?v=C0fsYiCI54g) — hosted by our Yearn Devs. To start writing your own strategy, please visit the [Yearn Brownie Mix](https://github.com/yearn/brownie-strategy-mix).
```
Comment (DarkGhost7) [inline docs/contributing/contribute.md:61]: in the link title `Yearn Vaults v2 Introduction`

Comment (DarkGhost7) [reply docs/contributing/contribute.md:61]: To start writing your own strategy, please check out our template [Yearn Brownie Mix]

File: docs/contributing/contribute.md [hunk 4]
```diff
@@ -85,15 +85,15 @@ When you decide to start, this reading material is going to come handy:
 - [Mint your own DAI](https://medium.com/ethereum-grid/forking-ethereum-mainnet-mint-your-own-dai-d8b62a82b3f7)
 - [Dev Docs](https://docs.yearn.finance/developers/v2/getting-started)
 
-Once you have the strategy ready. This will guide you further down the road.
+Once you have the strategy ready, this will guide you further down the road:
```
Comment (DarkGhost7) [inline docs/contributing/contribute.md:88]: Once you have (a|your) strategy ready, ...

File: docs/contributing/contribute.md [hunk 5]
```diff
@@ -85,15 +85,15 @@ When you decide to start, this reading material is going to come handy:
...
 
 - [Vault naming standard](https://docs.yearn.finance/developers/v2/naming-convention)
 - [Release process](https://docs.yearn.finance/developers/v2/DEPLOYMENT)
 - [Security check](https://docs.google.com/document/d/1hBKB73kJPQM71enrG8xoSFj7wxYmczUlgigyq2KkcTE/edit#heading=h.4ieoeyetfrxm)
 
 ##### Other reading material
 
-This is complement to the others but it's not directly related to creating strategies.
+This is complementary to the others, but it's not directly related to creating strategies.
```
Comment (DarkGhost7) [inline docs/contributing/contribute.md:96]: ... to the `above`, but ...

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]
Check out my suggestions and lmk what you think

File: docs/contributing/contribute.md (hunk 1 above)
Comment (MarcoWorms) [inline docs/contributing/contribute.md:55]:
```suggestion
You can see our progress on our [Github Project Board](https://contribute.yearn.farm/good-first-issue.html), and you can immediately begin contributing where you see fit. We recommend checking out issues tagged as `help wanted` and recommend sticking to the respective repository's coding guidelines (linting, formatting, PRs, etc). It's that simple 🙂. You can also join us on the #dev channel in [Discord](https://discord.gg/w89vVxeV9h).
```

File: docs/contributing/contribute.md (hunk 2 above)
Comment (MarcoWorms) [reply docs/contributing/contribute.md:59]:
```suggestion
If you want an overview of how our vaults work, please see the schema and accompanying descriptions in the [Vaults Overview](https://docs.yearn.finance/getting-started/products/yvaults/overview).
```

File: docs/contributing/contribute.md (hunk 4 above)
Comment (MarcoWorms) [reply docs/contributing/contribute.md:88]:
```suggestion
Once you have your strategy ready, this will guide you further down the road:
```

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Pull request merged**